src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/AsymmetricAlgorithm.cs

```diff
@@ -2,18 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Diagnostics;
-
 namespace System.Security.Cryptography
 {
     public abstract class AsymmetricAlgorithm : IDisposable
     {
         protected int KeySizeValue;
         protected KeySizes[] LegalKeySizesValue;
 
-        protected AsymmetricAlgorithm()
+        protected AsymmetricAlgorithm() { }
+
+        public static AsymmetricAlgorithm Create()
+        {
+            return Create("System.Security.Cryptography.AsymmetricAlgorithm");
```

Comment:
I thought it would be good for readability, if someone want to use the workaround they would pass in that string to CryptoConfig.CreateFromName(string)